### DISCUSSION TITLE
IntelliSense for "Quick Start a Hello World project (C++, CMake)"

### DISCUSSION THREAD

**Discussion by nicreuss** [xpack/vscode-xpack-extension-ts#27] [Q&A]
I've been struggling to get IntelliSense to work correctly for the [quick start example](https://xpack.github.io/vscode/quick-start/). 
The build is successful but IntelliSense has errors. 

The following things are working:
- CMake generates `compile_commands.json` 
- `c_cpp_properties.json` is updated with the path for `compile_commands.json` 
- The local headers (in the ` include` directory) are indexed by IntelliSense
 
However, the system header `iostream` (line 15 in hello-world.cpp) is not found.

![image](https://user-images.githubusercontent.com/44685248/145712617-632557a1-7f56-4eb3-84eb-395c0e5e08b3.png)

The C/C++ extension give this output:
`Unable to resolve configuration with compilerPath: "C:\Users\nic\Desktop\gcc-test\xpacks\.bin\gcc"`

Any ideas about what I am doing wrong?

## Environment
OS: Windows 10
xPack extension version: v0.4.16
C/C++ extension version: v1.7.1
Xpm version: 0.10.8
Node and global npm packages managed using [Volta](https://volta.sh/)

**Comment by ilg-ul** [comment]
Hmmm... You are not doing anything wrong, it seems that the C++ extension cannot recognise the compiler installed via xpm. :-(

The reason I could not detect it before is that on my test machine I had a `cl.exe` installed, and VS Code picked it.

I'll further investigate.

**Comment by ilg-ul** [comment]
I created #28 to keep track of the bug.

**Reply by nicreuss** [reply]
Thank you for looking into the issue.
